recipe/meta.yaml

```diff
@@ -23,16 +23,15 @@ requirements:
     - toml
     - wheel
   run:
-    - python
-    - colorama  # [win]
-    - importlib_resources  # [py<37]
```

Comment:
`importlib_resources  # [py<37]` is still required, see https://github.com/tqdm/tqdm/blob/6791e8c5b3d6c30bdd2060c346996bfb5a6f10d1/setup.cfg#L81

Reply:
Thanks, added.

Reply:
great, made amendments